.github/workflows/ci.yml

```diff
@@ -74,6 +74,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Build frontend signed
+        if: github.event_name != 'pull_request'
```

Comment:
hm. but this would mean, this is also skipped if we do internal pull requests, like this one, right?
so if we want to debug the signing, we would have to merge it into the master, otherwise we can not test it?
have you seen `pull_request_target`? https://runs-on.com/github-actions/pull-request-vs-pull-request-target/

what do you think of the following: we change the event to `pull_request_target` (thus external prs can access our secrets) but activate that all external collaborations have to be manually approved:
https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/approving-workflow-runs-from-public-forks

![image](https://github.com/user-attachments/assets/80fc8f13-b3f8-41b6-872a-b76d1e41031c)

Reply:
You are right, I did not consider our PRs. Your solution works but will add us the additional task to manually validate the collaborators. 
What do you think about triggering the validation if the action does not come from a PR or if the PR comes from our own repo? 

(github.event_name != 'pull_request') || (github.event.pull_request.head.repo.full_name == github.repository)

Reply:
nice. i have also searched for something like that, but could not find it.
the worst thing that can happen with your solution is that the signed build fails, which is not a catastrophe, so let's go for it!